src/app/utils/tests/TestUtils.spec.ts

```diff
@@ -554,3 +554,14 @@ export function expectValidRoutingLink(element: DebugElement, fullPath: string,
     const expectedComponent: string = getComponentClassName(component);
     expect(routedToComponent).withContext('It should route to the expected component').toEqual(expectedComponent);
 }
+
+/**
+ * Checks that a promise resulted in a firestore 'permission-denied' error.
+ * Useful to test that permissions on firestore work as expected.
+ */
+export async function expectPermissionToBeDenied<T>(promise: Promise<T>): Promise<void> {
+    await promise.then(() => {
```

Comment:
monolinise this one ? Sinon bien que lawful, l'indentation a weird

Reply:
vrai, cette indentation du cul :D Par contre l'espacage pour faire plaisir au linter est chelou

Reply:
a = func;
b = otherFunc;
promise.then(a, b) 
Ou autre :p ?
C'est beau l'art

Reply:
oui très bonne idée, beaucoup plus lisible :D